app-guides/udp-and-tcp.html.markerb

```diff
@@ -176,6 +176,10 @@ Another thing that won't impact your code that you'll need to know about is that
 
 Every Fly.io app gets an IPv6 address, and that address will work fine for the TCP side of your app, for whatever that's worth.
 
+<div class="note icon">
+This does not apply to 6PN; UDP over 6PN works just as you would expect.
```

Comment:
Suggest adding the word "note", and maybe including a link to 6PN info and removing the "just as you would expect" since they might not know about 6PN yet? Something like this (or whatever wording you think works!):
```suggestion
**Note:** UDP does work over private [6PN networks](/docs/reference/private-networking/).
```

Reply:
```suggestion
**Note:** UDP does work over private [Fly IPv6 private networking](/docs/reference/private-networking/).
```

Reply:
sorry being a pain, but if you remove the first "private" and change "Fly" to "Fly.io", then you can go ahead and merge away!

Reply:
```suggestion
**Note:** UDP does work over [Fly.io IPv6 private networking](/docs/reference/private-networking/).
```

Thanks!